<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Chronos Screen</string>
  </property>
  <widget class="Button" name="uiMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiBattery">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>

{:d}%</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiExternalMedia">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>{externalStorageIdentifier:s} ({percentFull:d}%)
{hoursSaved}:{minutesSaved}:{secondsSaved} / {hoursTotal}:{minutesTotal}:{secondsTotal}
</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiPlayAndSave">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Play &amp;&amp; Save
{ramUsed:1.2f}s RAM
{ramTotal:1.2f}s Avail.</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiRecord">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>    Record</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiResolution">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>{hRes:d}Ã—{vRes:d}
@{fps:d}fps</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiExposure">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>{name:s}
{exposure:s}</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiWhiteBalance">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>White
Balance</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiBlackCal">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Black Cal
Sensor</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="CheckBox" name="uiZebraStripes">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Zebra
Stripes</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>CheckBox::half</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>CheckBox::none</enum>
   </property>
  </widget>
  <widget class="CheckBox" name="uiFocusPeaking">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Focus
Peaking</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>CheckBox::half</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>CheckBox::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiFocusPeakingColor">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="customStyleSheet" stdset="0">
    <string>Button { border-width: 0px; }</string>
   </property>
  </widget>
  <widget class="InteractiveVideoArea" name="uiVideo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>801</width>
     <height>361</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="uiErrantClickCatcher" native="true">
   <property name="geometry">
    <rect>
     <x>799</x>
     <y>0</y>
     <width>1</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="Button" name="uiDebugD">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ðŸšª</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiDebugC">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiDebugB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiDebugA">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="uiBottomVerticalLine" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>301</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;</string>
   </property>
  </widget>
  <widget class="QWidget" name="uiBottomVerticalLine_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>1</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;</string>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>backdrop_2</zorder>
  <zorder>uiVideo</zorder>
  <zorder>uiMenu</zorder>
  <zorder>uiBattery</zorder>
  <zorder>uiExternalMedia</zorder>
  <zorder>uiPlayAndSave</zorder>
  <zorder>uiRecord</zorder>
  <zorder>uiResolution</zorder>
  <zorder>uiExposure</zorder>
  <zorder>uiWhiteBalance</zorder>
  <zorder>uiBlackCal</zorder>
  <zorder>uiZebraStripes</zorder>
  <zorder>uiFocusPeaking</zorder>
  <zorder>uiFocusPeakingColor</zorder>
  <zorder>uiErrantClickCatcher</zorder>
  <zorder>uiDebugD</zorder>
  <zorder>uiDebugC</zorder>
  <zorder>uiDebugB</zorder>
  <zorder>uiDebugA</zorder>
  <zorder>uiBottomVerticalLine</zorder>
  <zorder>uiBottomVerticalLine_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>check_box</header>
  </customwidget>
  <customwidget>
   <class>InteractiveVideoArea</class>
   <extends>QWidget</extends>
   <header>interactive_video_area</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiMenu</tabstop>
  <tabstop>uiBattery</tabstop>
  <tabstop>uiExternalMedia</tabstop>
  <tabstop>uiPlayAndSave</tabstop>
  <tabstop>uiRecord</tabstop>
  <tabstop>uiDebugA</tabstop>
  <tabstop>uiDebugB</tabstop>
  <tabstop>uiDebugC</tabstop>
  <tabstop>uiDebugD</tabstop>
  <tabstop>uiResolution</tabstop>
  <tabstop>uiExposure</tabstop>
  <tabstop>uiWhiteBalance</tabstop>
  <tabstop>uiBlackCal</tabstop>
  <tabstop>uiZebraStripes</tabstop>
  <tabstop>uiFocusPeaking</tabstop>
  <tabstop>uiFocusPeakingColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
